README.md

```diff
@@ -20,13 +20,25 @@ This module adds a new exec provider capable of executing PowerShell commands.
 
 ## Module Description
 
-Puppet provides a built-in `exec` type that is capable of executing commands. This module adds a `powershell` provider to the `exec` type,  which enables `exec` parameters, listed below. This module is particularly helpful if you need to run PowerShell commands but don't know the details about how PowerShell is executed, because you can run PowerShell commands in Puppet without the module.
+Puppet provides a built-in `exec` type that is capable of executing commands. This module adds a `powershell` and `pwsh` provider to the `exec` type,  which enables `exec` parameters, listed below. This module is particularly helpful if you need to run PowerShell commands but don't know the details about how PowerShell is executed, because you can run PowerShell commands in Puppet without the module.
 
 ## Setup
 
 ### Requirements
 
-This module requires PowerShell to be installed and the `powershell.exe` to be available in the system PATH.
+The `powershell` provider requires [Windows PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-5.1) to be installed and the `powershell.exe` to be available in the system PATH.
+
+The `pwsh` provider requires [PowerShell Core](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-6) to be installed and the `pwsh` to be available either in the system PATH or specified in the `path` parameter.
```

Comment:
Same suggestion as above:

"The `pwsh` provider requires you install [PowerShell Core] and make `pwsh` available either in the system PATH or specified in the `path` parameter."